tfjs-core/src/kernel_registry_test.ts

```diff
@@ -17,11 +17,11 @@
 
 import * as tf from './index';
 import {KernelBackend} from './index';
-import {ALL_ENVS, describeWithFlags} from './jasmine_util';
+import {ALL_ENVS, describeWithFlags, TestEnv} from './jasmine_util';
 import {KernelFunc, TensorInfo} from './kernel_registry';
 import {expectArraysClose} from './test_util';
 
-describeWithFlags('kernel_registry', ALL_ENVS, () => {
+describeWithFlags('kernel_registry', ALL_ENVS, (testEnv: TestEnv) => {
```

Comment:
Since this test set will call `tf.removeBackend` in most situations. How about adding below snippet here. Then we don't need to concern whether to add `await tf.setBackend(testEnv.backendName);` at the end of it.
```
  beforeAll(() => {
   // Comment the reason.
    await tf.setBackend(testEnv.backendName);
  });
```

Reply:
Ah, Just realize that it may only be called once. But we need to call it before each of iteration. Please ignore this comment.

Reply:
This is a good idea. We can use `afterEach` instead of `beforeAll`, and it will run after each test.